### PR TITLE
Tkt 00129 - secondary content area on front page

### PR DESCRIPTION
**Pull request by Android63**
Displays 2 latest posts in sidebar as shown below:

screen shot 2018-05-04 at 15 10 20

### CODE REVIEW

File: inc/template-tags.php [hunk 1]
```diff
@@ -37,48 +37,32 @@ function nightingale_wp_entry_footer() {
 
 	?>
 	<div class="o-layout">
-		<div class="o-layout__item u-9/12@lg">
+		<div class="o-layout__item u-6/12@lg">
```
Comment (cehwitham) [inline inc/template-tags.php]: Is this being used in more than the home page secondary? Does this have a knock on effect for post listings?

File: front-page.php [hunk 2]
```diff
@@ -34,11 +29,11 @@
 
 		</main><!-- #main -->
 	</div><!-- #primary -->
-<?php get_template_part( 'template-parts/content', 'secondary' ) ?>	
-<?php get_sidebar(); ?>
+	<?php get_template_part( 'template-parts/front-page', 'secondary' );?>
```
Comment (cehwitham) [inline front-page.php]: Do we need to maintain separate secondary content areas? Is this to switch out the flex content for the post listing?

File: inc/widgets.php [hunk 3]
```diff
@@ -33,6 +33,17 @@ function nightingale_wp_widgets_init() {
 		'after_title'   => '</h2>',
 	) );
 
+	// Front page sidebar (AKA secondary content area) widget area
```
Comment (cehwitham) [inline inc/widgets.php]: Do we still need this widget area? I thought we'd switched to just list the latest posts and not have it as widgets?

File: template-parts/content-excerpt.php [hunk 4]
```diff
@@ -27,8 +27,14 @@
 	<div class="c-article__content">
 		<?php
 
-			// Advanced Custom Fields content
-			get_template_part( 'template-parts/acf', 'flex' );
+			// Don't show post content if it's in a listing page
+			if ( is_single() ) {
+					// Advanced Custom Fields content
+					get_template_part( 'template-parts/acf', 'flex' );
```
Comment (cehwitham) [inline template-parts/content-excerpt.php]: Do we need to call the full ACF Flex if we're just trying out output the excerpt?

File: inc/template-tags.php (hunk 1 above)
Comment (Android63) [reply inc/template-tags.php]: Yes, it's used on all posts but there's no knock-on effect as there's still room for a comment count on the left and "read more" on the right.

File: inc/widgets.php (hunk 3 above)
Comment (Android63) [reply inc/widgets.php]: Well spotted Chris! I have removed this code.

File: front-page.php (hunk 2 above)
Comment (Android63) [reply front-page.php]: Yes. Content secondary shows flex content, which is used on other pages, but the home page displays the latest 2 blog posts.

**Comment by Android63** [comment]
Could you review this again please @cehwitham?

**Pull request merged**